<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <item row="5" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSpeedReadingSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>SpeedReading</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="speedGroupBox">
         <property name="title">
          <string>Speed</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="numberOfWordsLabel">
            <property name="text">
             <string>Number of Words:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="numberOfWordsSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="wordsPerMinuteLabel">
            <property name="text">
             <string>Words per minute:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="numberGroupingCheckBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If selected it tries not to split numbers, such as &quot;10 000&quot; into &quot;10&quot; and &quot;000&quot;. It'll remain as &quot;10 000&quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Number grouping</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="stallAtIndentionsCheckBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If an indention occurs, the SpeedReading will be stalled for a moment.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Stall at indentions</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="displayLongerWordsCheckBox">
            <property name="text">
             <string>Display longer words longer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="wordsPerMinuteSpinBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="wordLengthLabel">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Words longer than the given length, will be displayed longer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Word length:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="wordLengthSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="appearanceGroupBox">
         <property name="title">
          <string>Appearance</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="fontFamilyLabel">
            <property name="text">
             <string>Font Family:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="changeTextBackgroundColorPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="fontSizeSpinBox">
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="changeTextColorPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QFrame" name="linesColorFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QFrame" name="textBackgroundColorFrame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="textBackgroundColorLabel">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Background color of the area around the displayed text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Text background color:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="textColorLabel">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color of the displayed text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Text color:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="linesColorLabel">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color of the lines, which will be atop and below the displayed text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Lines color:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="fontFamilyComboBox"/>
          </item>
          <item row="5" column="1">
           <spacer name="appearanceSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="changeLinesColorPushButton">
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="fontSizeLabel">
            <property name="text">
             <string>Font size:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFrame" name="textColorFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWordsSettings">
      <attribute name="title">
       <string>Words</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutWordsSettings">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="wordsTableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>Word</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Stopword</string>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;It is a word, where the SpeedReading stops and you can review your stuff, which you have already speedread and what is going to come.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Breakword</string>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;It is a word, where it will only display the unspeedread words before the particularly word and the breakword.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Delayword (ms)</string>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;It is a word, where it will display the segment containing the delayword longer corresponding to the given value in ms. Negative values are not allowed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="wordLineEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="wordAddButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="wordDeleteButtons">
         <item>
          <widget class="QPushButton" name="wordDeleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="wordDeleteAllButton">
           <property name="text">
            <string>Delete all</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="wordDeleteButtonsVerticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>numberOfWordsSpinBox</tabstop>
  <tabstop>wordsPerMinuteSpinBox</tabstop>
  <tabstop>numberGroupingCheckBox</tabstop>
  <tabstop>stallAtIndentionsCheckBox</tabstop>
  <tabstop>displayLongerWordsCheckBox</tabstop>
  <tabstop>wordLengthSpinBox</tabstop>
  <tabstop>fontSizeSpinBox</tabstop>
  <tabstop>fontFamilyComboBox</tabstop>
  <tabstop>changeTextColorPushButton</tabstop>
  <tabstop>changeTextBackgroundColorPushButton</tabstop>
  <tabstop>changeLinesColorPushButton</tabstop>
  <tabstop>wordsTableWidget</tabstop>
  <tabstop>wordLineEdit</tabstop>
  <tabstop>wordAddButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
